app/controllers/chat_channels_controller.rb

```diff
@@ -240,7 +240,7 @@ def render_chat_channel
   end
 
   def send_open_notification
-    adjusted_slug = if @chat_channel.group?
+    adjusted_slug = if @chat_channel.type?("group")
```

Comment:
I still think `.group?` is clearer than `.type?(group)`. Is there are any naming conflict with keeping it `.group?`?

Reply:
I'll remove the commit about these methods changes.